datalad_registry/__init__.py

```diff
@@ -1,5 +1,12 @@
-__version__ = "0.0.0"
+import sys
 
 from celery import Celery
 
+if sys.version_info[:2] < (3, 8):
+    from importlib_metadata import version
+else:
+    from importlib.metadata import version
```

Comment:
just a note to myself: I thought to just make 3.8 to be minimal version , but since it is a datalad extension, we better stay aligned with versions of python datalad release supports. ATM it is still 3.6 (which EOLs this Dec)